lib/engines/content_block_manager/app/models/content_block_manager/content_block/edition.rb

```diff
@@ -30,6 +30,13 @@ def render
           details:,
         ).render
       end
+
+      def add_object_to_details(object_type, body)
```

Comment:
will we be able to do this to editions that are already technically `published`? the workflow state transitions aren't totally clear to me since we added the draft option.

Reply:
Yeah, this will work because we're updating the state directly instead of using the transformers